app/src/main/java/com/github/sdp/mediato/ui/NewItemFragment.java

```diff
@@ -138,13 +141,25 @@ public void addItem(View view) {
         if (reviewText.getText().length() > MAX_REVIEW_LENGTH) {
             requireActivity().runOnUiThread(() -> errorTextView.setText(String.format(Locale.ENGLISH, "Exceeded character limit: %d", MAX_REVIEW_LENGTH)));
         } else {
+            // Switch back to the profile page
+            // Forward the current arguments to the profile page (should be username and the name of the collection to add to)
             Intent intent = new Intent(getActivity(), MainActivity.class);
             String username = requireActivity().getIntent().getStringExtra("username");
+            String collectionName = getArguments().getString("collection");
             intent.putExtra("username", username);
             Review review = new Review(username, media,
-                    Integer.parseInt(ratingIndicator.getText().toString()), reviewText.getText().toString());
-            intent.putExtra("review", new Gson().toJson(review));
-            startActivity(intent);
+                Integer.parseInt(ratingIndicator.getText().toString()), reviewText.getText().toString());
+           /* intent.putExtra("review", new Gson().toJson(review));
```

Comment:
Do we need to keep this commented code ?

Reply:
Nope I removed it